reference/src/layout/enums.md

```diff
@@ -351,90 +351,76 @@ enum Enum1<T> {
 }
```

Comment:
Can you edit (further up) the definition of "data carrying" to say "enums where at least one variant ..."? The current "Enums whose variants" is ambiguous IMO.